gradle.properties

```diff
@@ -9,3 +9,6 @@ kotlinVersion=1.6.10
 dokkaVersion=1.6.10
 freemarkerVersion=2.3.29
 pluginPublishVersion=0.15.0
+
+# MaxMetaspaceSize set for Dokka, remove with the update to 1.8.0
+org.gradle.jvmargs=-XX:MaxMetaspaceSize=1G
```

Comment:
Caught a random OOM, hopefully it's just a combination of Dokka 1.6 and modest CI defaults - unable to reproduce locally.

Will try removing it with the update to 1.8